### PR TITLE
Improvement: Add the sdk version name in tabBank. RD-19660

### PR DESCRIPTION
**Pull request by dhouha123**
- Add the sdk version name text in tab Bank

### CODE REVIEW

File: app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java [hunk 1]
```diff
@@ -55,6 +56,8 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     public void onViewCreated(final View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         FragmentManager childFragmentManager = getChildFragmentManager();
+        TextView versionName = view.findViewById(R.id.versionName);
+        versionName.setText("V."+Dimelo.getInstance().getVersionName());
```
Comment (waelba) [inline app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java]: Can you add localization support like iOS please?

FR:  "Version du SDK :  X.X.X"
DE, EN:  "SDK version: X.X.X"

Comment (waelba) [inline app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java]: ~~please use the same iOS name for `Dimelo.getInstance().getVersionName()` so we can rename it to `Dimelo.getInstance().getSdkVersion()`~~

No need to add a new sdk function to retrieve the version name you can use directly this code:

`String libVersionName = com.dimelo.dimelosdk.BuildConfig.VERSION_NAME` (package_name.BuildConfig.VERSION_NAME)

Comment (dhouha123) [reply app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java]: ok

File: app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public void onViewCreated(final View view, @Nullable Bundle savedInstanceState)
         super.onViewCreated(view, savedInstanceState);
         FragmentManager childFragmentManager = getChildFragmentManager();
         TextView versionName = view.findViewById(R.id.versionName);
-        versionName.setText("V."+Dimelo.getInstance().getVersionName());
+        versionName.setText(getString(R.string.rc_sdk_version)+com.dimelo.dimelosdk.BuildConfig.VERSION_NAME);
```
Comment (waelba) [inline app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java]:
```suggestion
        versionName.setText(getString(R.string.rc_sdk_version) + com.dimelo.dimelosdk.BuildConfig.VERSION_NAME);
```

**Comment by DrissTM** [comment]
@dhouha123 please update the PR title/description and take the last comments into account

**Comment by dhouha123** [comment]
I added the description and I changed the title

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="rc_sdk_version">SDK version : </string>
```
Comment (dhouha123) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="rc_sdk_version">SDK version: </string>
```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
-    <string name="update">Update</string>
+    <string name="app_name" translatable="false">Demo App</string>
+    <string name="welcome_to_dimelo" translatable="false">Dimelo Mobile demo App</string>
+    <string name="how_to" translatable="false">Tap "Open Full Screen" to see the default Dimelo Mobile integration.</string>
+    <string name="telecom_welcome" translatable="false">Upgrade your plan, subscribe to services and contact us with just a tap.</string>
+    <string name="bank_welcome" translatable="false">Consult your accounts, transfer money and get in touch with just a tap.</string>
+    <string name="enable_thread" translatable="false">Enable Threads:</string>
+    <string name="user_identifier_text" translatable="false">Choose a user identifier:</string>
+    <string name="update" translatable="false">Update</string>
+    <string name="rc_sdk_version">SDK version : </string>
```
Comment (dhouha123) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="rc_sdk_version">SDK version: </string>
```

**Comment by waelba** [comment]
LGTM

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,12 +1,13 @@
 <resources>
-    <string name="app_name">Demo App</string>
-    <string name="welcome_to_dimelo">Dimelo Mobile demo App</string>
-    <string name="how_to">Tap "Open Full Screen" to see the default Dimelo Mobile integration.</string>
-    <string name="telecom_welcome">Upgrade your plan, subscribe to services and contact us with just a tap.</string>
-    <string name="bank_welcome">Consult your accounts, transfer money and get in touch with just a tap.</string>
-    <string name="enable_thread">Enable Threads:</string>
-    <string name="user_identifier_text">Choose a user identifier:</string>
-    <string name="update">Update</string>
+    <string name="app_name" translatable="false">Demo App</string>
```
Comment (DrissTM) [inline app/src/main/res/values/strings.xml:2]: is there any reason you added this `translatable="false"` everywhere?

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1,12 +1,13 @@
...
+    <string name="welcome_to_dimelo" translatable="false">Dimelo Mobile demo App</string>
```
Comment (DrissTM) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="welcome_to_dimelo" translatable="false">RingCentral Engage Messaging Mobile demo App</string>
```

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -1,12 +1,13 @@
...
+    <string name="how_to" translatable="false">Tap "Open Full Screen" to see the default Dimelo Mobile integration.</string>
```
Comment (DrissTM) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="how_to" translatable="false">Tap "Open Full Screen" to see the default RingCentral Engage Messaging integration.</string>
```

**Review by DrissTM — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (dhouha123) [reply app/src/main/res/values/strings.xml:2]: no just is the name of the application, you don't need to translate

Comment (DrissTM) [reply app/src/main/res/values/strings.xml:2]: I mean you added `translatable="false"` in every string in this file, what is the reason for this change?

Comment (dhouha123) [reply app/src/main/res/values/strings.xml:2]: translatable="false" we don't need to translate all this string even if we don't have the file of traduction

**Review by DrissTM — APPROVED** [review]
LGTM :+1:

**Pull request merged**